members/admin.py

```diff
@@ -451,17 +490,14 @@ def get_queryset(self, request):
         return qs.filter(activity__department__adminuserinformation__user=request.user)
 
     fieldsets = (
-        (None, {
-        'description' : '<p>Invitationer til en aktivitet laves nemmere via "person" oversigten. Gå derind og filtrer efter f.eks. børn på venteliste til din afdeling og sorter efter opskrivningsdato, eller filter medlemmer på forrige sæson. Herefter kan du trykke på personen og tilføje invitationer til personens invitations liste.</p>',
-
-        'fields' : (
-        'person',
-        'activity',
-        'invite_dtm',
-        'expire_dtm',
-        'rejected_dtm'
-        )
-    }),)
+        (
+            None,
+            {
+                'description': '<p>Invitationer til en aktivitet laves nemmere via "person" oversigten. Gå derind og filtrer efter f.eks. børn på venteliste til din afdeling og sorter efter opskrivningsdato, eller filter medlemmer på forrige sæson. Herefter kan du trykke på personen og tilføje invitationer til personens invitations liste.</p>',
+                'fields': ('person', 'activity', 'invite_dtm', 'expire_dtm', 'rejected_dtm')
```

Comment:
Jeg kan ikke helt find en god regel for om tuples og lists skal være udfoldet eller slået sammen